### ISSUE TITLE
Need a basic explanation on how to use this module.

### ISSUE THREAD

**Issue by ca3games** [FlutterTal/godot_ggpo#5]
There's not a basic documentation or tutorial or explanation on how to use this code.
I want to make a 2D figther, but I need some help on how to run the simulation in the resync step where I need to simulate again the other frames.

**Comment by FlutterTal** [comment]
Well for now, I'm more worried about knowing if my module really works as intended rather than making a documentation or tutorial of this module. But yeah, that is definitely something I'm gonna do later on.
While waiting, if you wanna try and use the module, you can. And if you wanna know how to use it, the GGPO repo should be enough to help you with that.
https://github.com/pond3r/ggpo

**Comment by ca3games** [comment]
@FlutterTal Yeah, I understand, but again, I could work it out, what I kind of not understand yet is how to do the resync step where I have to simulate the other frames.
I still don't know how to run multiple frames on godot on a single frame.

**Comment by MIfeanyi** [comment]
@FlutterTal Do you know how to do the actual rollback in Godot?

**Comment by MIfeanyi** [comment]
This plugin provides a very good setup use case tutorial which will help a lot of people with this project: https://github.com/Lexpartizan/Godot_sky_shader
